datahub-dao/src/main/java/com/linkedin/datahub/util/CorpUserUtil.java

```diff
@@ -33,10 +32,14 @@ public static CorpuserUrn toCorpUserUrn(@Nonnull String urnStr) throws URISyntax
     @Nonnull
     public static User toCorpUserView(CorpUser corpUser) {
         User user = new User();
-        user.setEmail(corpUser.getInfo().getEmail());
-        user.setName(corpUser.getInfo().getFullName());
         user.setUserName(corpUser.getUsername());
-        user.setPictureLink(corpUser.getEditableInfo().getPictureLink().toString());
+        if (corpUser.hasInfo()) {
```

Comment:
I think `corpUser.getInfo() != null` is a safer check. Same for editable info.

Reply:
Why is that? can the writers set null values for optional fields ?

Reply:
null value can be set with `IGNORE_NULL` mode?

Reply:
Makes sense. Will change this for more safer checks.